### PR TITLE
Setup.py and README created, basic bin file with argument parser and test script

### PR DESCRIPTION
**Pull request by joshglens**
Setup.py created
README updated to include venv and global install
Basic bin/ringling-cli with argument parser create
ringling/test_script.py created to test accessibility of folder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,40 @@
...
+```
+
+## Option 1. Install in Virtual Environment
+If you would prefer to use a virtual environment instead of installing system wide, you can:
+
+#### Linux:
+```bash
+$ cd ringling-cli
+$ python3 -m venv venv
+$ source venv/Scripts/activate
```
Comment (rnowling) [inline README.md:21]: If you're on Linux, MacOS, or similar, it's `source venv/bin/activate`

File: bin/ringling-cli [hunk 2]
```diff
@@ -0,0 +1,34 @@
+"""
+Copyright 2023 Joshua Goldshteyn
```
Comment (rnowling) [inline bin/ringling-cli]: Change the author to `MSOE DISE Project`

File: bin/ringling-cli [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+                        required=False)
+
+    return parser.parse_args()
+
+if __name__ == "__main__":
+    args = parseargs()
+    print_test()
+    print(args)
+    print(args.action)
+    print(type(args))
```
Comment (rnowling) [inline bin/ringling-cli]: Missing EOL character?

File: ringling/test_script.py [hunk 4]
```diff
@@ -0,0 +1,2 @@
+def print_test():
+    print("Access to test_script.py in ringling directory successful")
```
Comment (rnowling) [inline ringling/test_script.py]: Missing EOL character?

File: ringling/test_script.py [hunk 5]
```diff
@@ -0,0 +1,2 @@
+def print_test():
```
Comment (rnowling) [inline ringling/test_script.py:17]: Add license header

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,29 @@
+"""
+Copyright 2023 Joshua S. Goldshteyn
```
Comment (rnowling) [inline setup.py]: Change the author to `MSOE DISE Project`

File: setup.py [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from setuptools import setup
+
+setup(name="ringling-cli",
+      version=0.1,
+      description="Client for Ringling",
+      author="Joshua Goldshteyn",
```
Comment (rnowling) [inline setup.py]: Change the author to `MSOE DISE Project`

File: setup.py [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+      author_email="jsgoldshteyn@gmail.com",
```
Comment (rnowling) [inline setup.py]: Change to nowling@msoe.edu

File: setup.py [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+      license="Apache License, Version 2.0",
+      zip_safe=False,
+      packages=["ringling"],
+      python_requires=">=3.8",
+      #install_requires = ["numpy>=0.19.1", "scipy>=0.19.1", "matplotlib", "seaborn", "sklearn", "joblib", "pandas", "mmh3"],
```
Comment (rnowling) [inline setup.py]: Don't commit commented-out code

**Review by rnowling — CHANGES_REQUESTED** [review]
Great first PR!  You should add an `__init__.py` file to the `ringling` folder so that it seen as a packaged.  The file only needs to contain the license header -- nothing else.  Thanks!

File: README.md (hunk 1 above)
Comment (joshglens) [reply README.md:21]: When I install on Windows it doesn't create a venv/bin folder. Would it make sense to create separate Windows and Linux/MacOS instructions? Or just assume that the end user will figure it out?

Comment (rnowling) [reply README.md:21]: I would say leave it be for now

**Review by rnowling — APPROVED** [review]

**Pull request merged**